src/Identity/Controllers/SsoController.cs

```diff
@@ -86,9 +95,11 @@ public async Task<IActionResult> PreValidate(string domainHint)
         public async Task<IActionResult> Login(string returnUrl)
         {
             var context = await _interaction.GetAuthorizationContextAsync(returnUrl);
+            var query = Request.Query;
 
             var domainHint = context.Parameters.AllKeys.Contains("domain_hint") ?
                 context.Parameters["domain_hint"] : null;
+            var ssoToken = context.Parameters["ssoToken"];
```

Comment:
You had a previous pattern where this string (or it's prior revision) was stored in a const, `"ssoToken"`. Since this is re-used in `Startup.cs`, in the SSO `AccountController.cs`, and here, it would seem prudent to return to that pattern vs. using embedded strings here... It's not necessary, but does make refactoring easier in the future if that was required. Perhaps in `TokenGenerator`?

Reply:
I see you have that const in there, but it's not used here.
```suggestion
            var ssoToken = context.Parameters[SsoToken.TokenName];
```